docs/next.config.js

```diff
@@ -19,7 +19,10 @@ const isDeployPreview = process.env.PULL_REQUEST === 'true';
 const buildOnlyEnglishLocale = isDeployPreview && !l10nPRInNetlify && !vercelDeploy;
 
 const staging =
-  process.env.REPOSITORY_URL === undefined || /mui\/material-ui$/.test(process.env.REPOSITORY_URL);
+  process.env.REPOSITORY_URL === undefined ||
+  // The linked repository url comes from https://app.netlify.com/sites/material-ui/settings/deploys
+  /mui-org\/material-ui$/.test(process.env.REPOSITORY_URL);
```

Comment:
I would have expected 😁

```suggestion
  /mui\/material-ui$/.test(process.env.REPOSITORY_URL);
```

It doesn't seem to work https://master--material-ui.netlify.app/joy-ui/getting-started/quick-start/

Reply:
For context, I was trying to share the WIP on Joy, for a PM on Twitter

<img width="566" alt="Screenshot 2022-05-24 at 00 59 20" src="https://user-images.githubusercontent.com/3165635/169917914-8561cc71-9957-46a1-a187-d97617340532.png">

Reply:
I think my change is correct. https://deploy-preview-32866--material-ui.netlify.app/joy-ui/react-button/

Here is the repo URL in netlify, it is `mui-org`:
<img width="633" alt="Screen Shot 2565-05-24 at 07 07 10" src="https://user-images.githubusercontent.com/18292247/169923285-722f6d81-e1f8-4549-aa63-6b195f006390.png">


I have to fix this page, not sure how it came https://deploy-preview-32866--material-ui.netlify.app/joy-ui/getting-started/quick-start/

Reply:
Ah perfect! I have followed this link:

<img width="551" alt="Screenshot 2022-05-24 at 11 18 45" src="https://user-images.githubusercontent.com/3165635/169997229-859840fc-3e3b-4467-8268-afa6ec53040e.png">

Reply:
Thanks, it is fixed in https://github.com/mui/material-ui/pull/32883